### PR TITLE
Update pf packages to pf-2022-4

### PR DESCRIPTION
**Pull request by jcaianirh**

### CODE REVIEW

File: frontend/public/vendor.scss [hunk 1]
```diff
@@ -5,6 +5,7 @@
 @import "~openshift-logos-icon/style";
 @import "~@fortawesome/fontawesome-free/scss/fontawesome";
 @import "~@fortawesome/fontawesome-free/scss/solid";
+@import "~@patternfly/patternfly/patternfly-theme-dark";
```
Comment (mattnolting) [inline frontend/public/vendor.scss:8]: You'll need to import this after all other `@patternfly/patternfly/` import rules, so after  `@import "~@patternfly/patternfly/patternfly-charts";`

**Review by vikram-raj — COMMENTED** [review]
Verified the changes.

/lgtm

**Comment by jcaianirh** [comment]
QE Approver:
/assign @yapei

Docs Approver:
/assign @opayne1

PX Approver:
/assign @RickJWagner

Console Approver:
/assign @rhamilto

File: frontend/public/vendor.scss [hunk 2]
```diff
@@ -30,6 +30,6 @@
 // External dependency extensions
 @import "style/patternfly-sizing-extension";
 
-// Dark theme styles from patternfly
-// Temporarily using css instead of scss due to a PF issue: https://github.com/patternfly/patternfly/issues/4700
-@import '~@patternfly/patternfly/patternfly-theme-dark.css';
+// PF Dark theme to be imported after all other pf imports
+@import "~@patternfly/patternfly/patternfly-theme-dark";
```
Comment (rhamilto) [inline frontend/public/vendor.scss:34]: Hooray!

**Review by rhamilto — COMMENTED** [review]
/lgtm

**Comment by yapei** [comment]
tested on Firefox 97, Chrome 100 and Safari 14, no issues found, going to approve
/label qe-approved

**Pull request merged**